src/phoenix/experimental/evals/models/anthropic.py

```diff
@@ -139,8 +140,13 @@ def _generate_with_retry(self, **kwargs: Any) -> Any:
         @self.retry
         @self._rate_limiter.limit
         def _completion_with_retry(**kwargs: Any) -> Any:
-            response = self.client.completions.create(**kwargs)
-            return response.completion
+            try:
+                response = self.client.completions.create(**kwargs)
+                return response.completion
+            except self._anthropic.BadRequestError as e:
+                if "prompt is too long" in e.args[0]:
+                    raise PhoenixContextLimitExceeded("Maximum context length exceeded.")
```

Comment:
```suggestion
                    raise PhoenixContextLimitExceeded("Maximum context length exceeded.") from e
```

Suggestion: Use exception chaining for a more detailed exception message.